site/docs/components/landing/WalletDemo.tsx

```diff
@@ -61,10 +59,9 @@ function WalletDemo() {
             Wallet
           </WalletDropdownLink>
           <WalletDropdownBasename />
-          <WalletDropdownDisconnect />
```

Comment:
given that we are removing disconnect option, consider adding a warning like, "wallet address for demonstration purposes only. do not send funds to this address"